ios/brave-ios/Sources/AIChat/Components/AIChatView.swift

```diff
@@ -41,6 +41,9 @@ public struct AIChatView: View {
   @State
   private var feedbackToast: AIChatFeedbackToastType = .none
 
+  @State
+  private var feedbackPrivacyWarning: AIChatFeedbackToastPrivacyWarning = .none
```

Comment:
Should just be optional, no `none` case

Reply:
I dislike optional because you have to then unwrap the enum. But done anyway.